### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by PrateekCodes**

### CODE REVIEW

**Comment by kumarldh** [comment]
भाई ये तो किसी bot का इस्तेमाल कर के किया है| कोई सर पैर नहीं| merge भी हो गया... 👎

File: 1-js/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # जावास्क्रिप्ट की भाषा
 
-Here we learn JavaScript, starting from scratch and go on to advanced concepts like OOP.
+यहाँ हम सीखते हैं JavaScript, खरोंच से शुरू और उन्नत अवधारणाओं की तरह चलते हैं OOP.
 
-We concentrate on the language itself here, with the minimum of environment-specific notes.
+हम पर्यावरण-विशिष्ट नोटों की न्यूनतम के साथ ही यहां की भाषा पर ध्यान केंद्रित करते हैं।
```
Comment (Harendra1993) [inline 1-js/index.md:5]: It seems like you just use a translator tool. Translated sentence should be meaningful.

**Comment by theBstar** [comment]
First of all, thank you for investing your time to rise a PR. But there are quite a few issues with the translated version (your changes). Most of the sentences doesn't convey the meaning well. We can definitely do better. But I haven't seen any activity on the PR. Are you available to make the changes @PrateekCodes ?

**Comment by theBstar** [comment]
I am closing the PR due to no-activity. Please feel free to re-open if you would still be able to contribute.

**Pull request closed without merging**